frontend-web/webclient/app/UCloud/JobsApi.tsx

```diff
@@ -154,6 +154,8 @@ export interface OpenInteractiveSessionRequest {
     id: string;
     rank: number;
     sessionType: "WEB" | "VNC" | "SHELL";
+    target?: string | null;
```

Comment:
Can this really be undefined, string or null, or just two of the three?

Reply:
It can be all three.